### PR TITLE
[Example] Update faster rcnn doc

### LINKED ISSUE #7928: Cannot reproduce the results of faster-rcnn
For bugs or installation issues, please provide the following information.
The more information you provide, the more likely people will be able to help you.

## Environment info
Operating System: Debian GNU/Linux 8 (jessie)

Compiler: gcc & g++ (4.9.2) 

Package used (Python/R/Scala/Julia): Python

MXNet version: 0.11.1

Or if installed from source: 

MXNet commit hash (`git rev-parse HEAD`): 2372518e22ad329b4a6a61486144760350171a80

If you are using python package, please provide

Python version and distribution: 2.7 / anaconda

## Error Message:
Cannot reproduce the mAPs of faster-rcnn with resnet101+voc2017+voc2012. 
The mAP I got is ~0.69 which is far less than the claimed 0.79.

If I use the pre-trained model from `https://github.com/precedenceguo/mx-rcnn`, the mAPs is correct. Please also see [this issue](https://github.com/precedenceguo/mx-rcnn/issues/72).

## Minimum reproducible example
in the rcnn folder, `script/resnet_voc0712.sh 0`

### PR DESCRIPTION
**Pull request by stu1130**
## Description ##
The default setting will only use voc2007 to train the model. Add one more example to make it more clear.
Fixes #7928
### Essentials ###
Please feel free to remove inapplicable items for your PR.
- [x] The PR title starts with [MXNET-$JIRA_ID], where $JIRA_ID refers to the relevant [JIRA issue](https://issues.apache.org/jira/projects/MXNET/issues) created (except PRs with tiny changes)
- [x] Changes are complete (i.e. I finished coding on this PR)
- [x] All changes have test coverage:
- Unit tests are added for small changes to verify correctness (e.g. adding a new operator)
- Nightly tests are added for complicated/long-running ones (e.g. changing distributed kvstore)
- Build tests will be added for build configuration changes (e.g. adding a new build option with NCCL)
- [x] Code is well-documented: 
- For user-facing API changes, API doc string has been updated. 
- For new C++ functions in header files, their functionalities and arguments are documented. 
- For new examples, README.md is added to explain the what the example does, the source of the dataset, expected performance on test set and reference to the original paper if applicable
- Check the API doc at http://mxnet-ci-doc.s3-accelerate.dualstack.amazonaws.com/PR-$PR_ID/$BUILD_ID/index.html
- [x] To the my best knowledge, examples are either not affected by this change, or have been fixed to be compatible with this change

### Changes ###
- [x] Feature1, tests, (and when applicable, API doc)
- [x] Feature2, tests, (and when applicable, API doc)

## Comments ##
N/A

### CODE REVIEW

**Comment by anirudhacharya** [comment]
@mxnet-label-bot [pr-awaiting-review]

**Comment by harshp8l** [comment]
@mxnet-label-bot add [pr-awaiting-review]
